config/routes.rb

```diff
@@ -22,8 +22,9 @@ def matches?(request)
 
     constraints account: /[^\/?]+/ do
       constraints authenticator: /authn-?[^\/]*/, id: /[^\/?]+/ do
+        get '/authn-jwt/:service_id/:account/:id/status' => 'authenticate#authn_jwt_status'
```

Comment:
please look at Oren fix for the generic route to be more specific the authenticator is supports in order to prevent from the customer by mistake to send the following request `/authn-jwt/:service_id/:account/status` (without id) which leads to another code that we do not know his behavior